src/headset/oculus_mobile.c

```diff
@@ -483,6 +475,7 @@ void bridgeLovrInit(BridgeLovrInitData *initData) {
   // Copypaste the init sequence from lovrRun:
   // Load libraries
   L = luaL_newstate(); // FIXME: Just call main?
+  lua_atpanic(L, luax_custom_atpanic);
```

Comment:
I was getting panics for a while and was able to fix it in https://github.com/bjornbytes/lovr/commit/185bb8438e9a14f79bba40b9b8de3478d4157200.  Does changing `lovrSetErrorCallback` to error on `Lcoroutine` instead of `L` fix any of the panic issues you were having?  I'd sorta prefer it if panicking was not a normal error handling mechanism.

Reply:
I'm not currently getting panics; the panics were in lovr.draw and the rest of the patch fixes them. The problem is that without this patch a panic results in *no* debug output— actually, on oculus mobile it causes a crash in a different thread (because the main thread tries to crash and some internal render thread then crashes). So this is needed because without it, development on oculus mobile becomes very difficult (because the act of development can introduce panics, and this patch makes it possible to distinguish a panic from heap corruption).